openedx/core/djangoapps/notifications/base_notification.py

```diff
@@ -230,6 +230,22 @@
         'email_template': '',
         'filters': [FILTER_AUDIT_EXPIRED_USERS_WITH_NO_ROLE],
     },
+    'audit_access_expiring_soon': {
+        'notification_app': 'enrollments',
+        'name': 'audit_access_expiring_soon',
+        'is_core': True,
+        'info': '',
+        'non_editable': [],
+        'content_template': _('<{p}>Your audit access for <{strong}>{course_name}</{strong}> is expiring on '
+                              '<{strong}>{{api_trigger_properties.${audit_access_expiry}}}</{strong}>. '
```

Comment:
This placeholder will not work because template contains `{{api_trigger_properties.${audit_access_expiry}}}` and passed context contains `audit_access_expiry`